blog/web/blog.md

```diff
@@ -0,0 +1,143 @@
+
+# Improve traceability with the tmt web app
+
+The [tmt web][web] app is a simple web application that makes it
+easy to explore and share test and plan metadata without needing
+to clone repositories or run tmt commands locally.
+
+At the beginning, there was the following user story:
+
+    As a tester, I need to be able to link the test case(s)
+    verifying the issue so that anyone can easily
+    find the tests for the verification.
```

Comment:
I would recommend `>` instead of indentation. Indentation makes a literal block, which makes line breaks stand up. It's not a code block, it's a quote.